### PR TITLE
B3289

### PR DESCRIPTION
**Pull request by BirkAndMe**
https://platform.dandigbib.org/issues/3289

### CODE REVIEW

File: modules/ding_event/ding_event.features.inc [hunk 1]
```diff
@@ -40,5 +40,6 @@ function ding_event_node_info() {
       'help' => t('This is the heading usually displayed with a big font size (h1) on the top of the page.'),
     ),
   );
+  drupal_alter('node_info', $items);
```
Comment (cableman) [inline modules/ding_event/ding_event.features.inc]: Why is this drupal_alter added and is it related to features update?

File: modules/ding_event/ding_event.info [hunk 2]
```diff
@@ -123,5 +121,5 @@ features[variable][] = webform_node_ding_event
 features[views_view][] = ding_event
 features_exclude[field_base][field_place2book_tickets] = field_place2book_tickets
 features_exclude[field_instance][node-ding_event-field_place2book_tickets] = node-ding_event-field_place2book_tickets
+features_exclude[variable][node_options_ding_event] = node_options_ding_event
```
Comment (cableman) [inline modules/ding_event/ding_event.info]: Why have this been exclude?

File: modules/ding_news/ding_news.features.inc [hunk 3]
```diff
@@ -34,16 +34,9 @@ function ding_news_image_default_styles() {
...
-    'name' => 'ding_panorama_list_large_wide',
     'label' => 'ding_panorama_list_large_wide',
     'effects' => array(
       22 => array(
-        'label' => 'Manual Crop: Reuse cropped style',
-        'help' => 'Reuse a crop selection from another Manual Crop enabled image style.',
-        'effect callback' => 'manualcrop_reuse_effect',
-        'form callback' => 'manualcrop_reuse_form',
-        'summary theme' => 'manualcrop_reuse_summary',
-        'module' => 'manualcrop',
```
Comment (cableman) [inline modules/ding_news/ding_news.features.inc]: Why is this information removed?

File: modules/ding_news/ding_news.features.inc [hunk 4]
```diff
@@ -52,13 +45,6 @@ function ding_news_image_default_styles() {
         'weight' => 0,
       ),
       23 => array(
-        'label' => 'Scale',
-        'help' => 'Scaling will maintain the aspect-ratio of the original image. If only a single dimension is specified, the other dimension will be calculated.',
-        'effect callback' => 'image_scale_effect',
-        'dimensions callback' => 'image_scale_dimensions',
-        'form callback' => 'image_scale_form',
-        'summary theme' => 'image_scale_summary',
-        'module' => 'image',
```
Comment (cableman) [inline modules/ding_news/ding_news.features.inc]: Why is this information removed?

File: modules/ding_news/ding_news.features.inc [hunk 5]
```diff
@@ -87,5 +73,6 @@ function ding_news_node_info() {
       'help' => '',
     ),
   );
+  drupal_alter('node_info', $items);
```
Comment (cableman) [inline modules/ding_news/ding_news.features.inc]: See previous comment about this alter!

File: modules/ding_news/ding_news.info [hunk 6]
```diff
@@ -94,4 +93,4 @@ features[variable][] = scheduler_unpublish_enable_ding_news
 features[variable][] = webform_node_ding_news
 features[views_view][] = ding_news
 features[views_view][] = ding_node
-mtime = 1507797796
+features_exclude[variable][node_options_ding_news] = node_options_ding_news
```
Comment (cableman) [inline modules/ding_news/ding_news.info]: Why this exclude?

**Comment by cableman** [comment]
Why is the re-export of ding_event and ding_news needed to lock the front page feature?

File: modules/ding_event/ding_event.features.inc (hunk 1 above)
Comment (BirkAndMe) [reply modules/ding_event/ding_event.features.inc]: The new version of features add this.

File: modules/ding_event/ding_event.info (hunk 2 above)
Comment (BirkAndMe) [reply modules/ding_event/ding_event.info]: The "promote types to frontpage" uses this, and we don't want these settings to be overriden when the feature is reverted

File: modules/ding_news/ding_news.features.inc (hunk 3 above)
Comment (BirkAndMe) [reply modules/ding_news/ding_news.features.inc]: The new features version removes this, my guess is this is redundant information.

File: modules/ding_news/ding_news.features.inc (hunk 4 above)
Comment (BirkAndMe) [reply modules/ding_news/ding_news.features.inc]: The new features version removes this, my guess is this is redundant information.

File: modules/ding_news/ding_news.info (hunk 6 above)
Comment (BirkAndMe) [reply modules/ding_news/ding_news.info]: See comment on ding_event.info

File: modules/ding_event/ding_event.info (hunk 2 above)
Comment (cableman) [reply modules/ding_event/ding_event.info]: What is "promote types to frontpage"?

File: modules/ding_event/ding_event.features.inc (hunk 1 above)
Comment (cableman) [reply modules/ding_event/ding_event.features.inc]: Okay

File: modules/ding_news/ding_news.features.inc (hunk 3 above)
Comment (cableman) [reply modules/ding_news/ding_news.features.inc]: Okay

File: modules/ding_news/ding_news.features.inc (hunk 4 above)
Comment (cableman) [reply modules/ding_news/ding_news.features.inc]: okay

File: modules/ding_news/ding_news.features.inc (hunk 5 above)
Comment (cableman) [reply modules/ding_news/ding_news.features.inc]: okay

**Comment by cableman** [comment]
Reviewed and awaiting release

**Pull request merged**